open-metadata-implementation/adapters/open-connectors/repository-services-connectors/open-metadata-collection-store-connectors/docs/README.md

```diff
@@ -26,4 +26,77 @@ metadata in an existing metadata repository.
 * **[Overview of the Event Mapper Connector Interface](overview-of-the-event-mapper-connector-interface.md)** -
 a walk-through of the methods on the event mapper connector interface.  This connector enables
 events from an existing metadata repository that indicate that metadata is changing, to be passed
-to the OMRS so it can be distributed to the other metadata repositories who are members of the same cohort.
\ No newline at end of file
+to the OMRS so it can be distributed to the other metadata repositories who are members of the same cohort.
+
+## Getting Started Implementing a New Connector
+
+As implied by the overview above, a logical place to start in implementing a new
+Open Metadata Collection Store Connector is by implementing the OMRS interface.
+
+This requires implementing:
+
+- an `OMRSRepositoryConnectorProvider` specific to your connector, which extends `OMRSRepositoryConnectorProviderBase`.
+    This really only involves defining the connector class and setting this in the constructor.
+- an `OMRSRepositoryConnector` specific to your connector, which extends `OMRSRepositoryConnector`. Most likely the
+    main logic of this class will be implemented by overriding the `initialize()` method of the base class,
+    and putting into this any logic for initializing the connection: for example, connecting to an underlying
+    database, or starting a REST API session, etc. (Similarly, override the `disconnect()` method to properly
+    cleanup / close such resources.)
+- an OMRSMetadataCollection specific to your repository, which extends `OMRSMetadataCollectionBase`. Ideally your
+    implementation should override each of the methods defined in the base class, but a logical place to start is
+    likely to be the `getEntityDetail()` method that retrieves an entity by its GUID.
+
```

Comment:
How about introducing the repository helper and validator since they simplify the implementation.

Reply:
I seem to have gotten away with just re-using the repository helper and validator I get from the base class directly (without overriding anything there) (?)  I've probably skipped something I need to do (eventually), but maybe not needed just to get started with the implementation of such a connector?  (Needs to be there to be conformant, but not critical just to get started?)

Reply:
That is the right appropach - you do not need to override their methods - just make use of their function :)